save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/vuln/CreateVulnerabilityView.kt

```diff
@@ -86,19 +87,15 @@ val createVulnerabilityView: VFC = VFC {
         setUserOrganizations(organizations)
     }
 
-    val fetchProject: (VulnerabilityProjectDto) -> Unit = { project ->
-        setVulnerability {
-            it.copy(projects = it.projects.plus(project))
```

Comment:
This can no longer be deleted. `vulnerabilityProjectWindow` returns an `affected` object
```
setCosv { cosv ->
            cosv.copy(
                    affected = cosv.affected.plus(project)
                )
        }
```

Reply:
will add back

Reply:
will be done in https://github.com/saveourtool/save-cloud/issues/2748